### ISSUE TITLE
[Feature Request] Show more info about user in the profile tab.

### ISSUE THREAD

**Issue by zerobun** [lotusprey/otraku#127]
Instead of just trying to show the user bio in the profile tab, maybe add the ability to show the user statistics? Making the user bio compact (have to click on it to open the whole bio and see it) and under it having the user statistics displayed would be more informative and quick.

I got the idea from from Al-chan

Here is a screenshot of what it could look like: 
![Screenshot_20240112095334](https://github.com/lotusprey/otraku/assets/127630316/3c5eeda1-8a1d-4281-ad3b-5966f68161f5)

Really love the apl BTW been using it for a long time.

Hope u consider it, thanks.

**Comment by lotusprey** [comment]
I would maybe put the activity history below the bio, since I don't know what other placement would be proper for it and statistics already have their own page.

**Comment by lewdpatriot** [comment]
> I would maybe put the activity history below the bio, since I don't know what other placement would be proper for it and statistics already have their own page.

Um, I think putting the stastics section there is a better idea instead of a separate section because (I think) most people click on profile to know their stastics most of the time. Everything else should be in separate pages.

I originally wanted to make a request on this myself. Please consider it, Thanks.

**Comment by lotusprey** [comment]
Why would most people click on profiles to look at statistics? I find it more reasonable to argue they would look at acitvities or that person's anime/manga lists.
My thinking is that the activity history makes the least sense to have its own page since it brings too little information by itself. It is also something that updates each day, while statistics change at a slower pace. Now, in the future, I may pull the most basic of stats below the activity history, at the bottom - I don't have a concrete plan yet.

**Comment by lewdpatriot** [comment]
> I may pull the most basic of stats at the bottom 

That's absolutely fine by me. The basic stats shown in the image minus 'mean score' probably?

My main point is that the page feels very empty right now. It would be nice to have something (anything) useful on there.

**Comment by dlsf** [comment]
There is a lot of useful information on there when users set their bio. Minimal stats might be optimal there

**Comment by lewdpatriot** [comment]
> There is a lot of useful information on there when users set their bio. Minimal stats might be optimal there

You're right. I completely forgot because I don't have a bio.

**Comment by zerobun** [comment]
> I would maybe put the activity history below the bio, since I don't know what other placement would be proper for it and statistics already have their own page.

Yes I agree, and maybe if you add the favorites tab after that would be nice for people who visit and look at someone's profile, and they get an idea of what anime they like at a glance.

And the followed option, to be next to the username as numbers of followers and following would be more compact too.

I have made a rough drawing over my own profile showcasing how it could possibly look like.


![Screenshot_20240113130004](https://github.com/lotusprey/otraku/assets/127630316/e25b59a7-a56e-4938-ac81-6f816eb69c30)
